web/packages/teleport/src/components/Authenticated/Authenticated.tsx

```diff
@@ -58,7 +58,7 @@ const Authenticated: React.FC<PropsWithChildren> = ({ children }) => {
       }
 
       try {
-        await session.validateCookieAndSession();
+        await session.fetchStatus();
```

Comment:
i think we need to revert this, or perhaps send `rememberLocation = true` to `fetchStatus` and remove all the code below this line.

I think it was done this way just so we can say `session.logout(rememberLocation)`, but in `fetchStatus` i thnk we logout before we can say remember location